src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3ConnectionException.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace System.Net.Http
+{
+    internal sealed class Http3ConnectionException : Http3ProtocolException
```

Comment:
Same comments as elsewhere on exceptions... if they can escape to user code, let's ensure they're [Serializable] and have a deserialization ctor.